src/components/Cart/CartItems.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
+import "./Cart.css";
+export const CartItems = () => {
+  const { cartState, cartDispatch } = useCart();
+  const { wishlistState, wishlistDispatch } = useWishlist();
+
+  return (
+    <>
+      <div className="cart-product-container">
+        {cartState.map((product) => {
+          return (
+            <div className="cart-card" key={product._id}>
+              <div className="cart-img-container">
+                <img
+                  src={product.img}
+                  alt={product.title}
+                  className="cart-img"
+                />
+                <i
+                  className="fas fa-trash-alt delete"
+                  onClick={() =>
+                    cartDispatch({
+                      type: "REMOVE_FROM_CART",
+                      payload: product,
+                    })
+                  }
+                ></i>
+              </div>
+
+              <div className="card-content-hrz">
+                <h4 className="card-title-hrz">{product.title}</h4>
+                <p className="card-seller-hrz">by {product.author}</p>
+
+                <div className="quantity">
+                  <button
+                    className="increase-quantity"
+                    onClick={() =>
```

Comment:
- Can we extract these onClick handlers to separate functions themselves instead of using inline anonymous functions.
- That will help in readability and also in usability if you want to dispatch the same action by another component.